backend/app/store/service/title.go

```diff
@@ -38,33 +41,48 @@ func NewTitleExtractor(client http.Client) *TitleExtractor {
 }
 
 // Get page for url and return title
-func (t *TitleExtractor) Get(url string) (string, error) {
+func (t *TitleExtractor) Get(pageURL string) (string, error) {
+	// parse domain of the URL and check if it's in the allowed list
+	u, err := url.Parse(pageURL)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse url %s: %w", pageURL, err)
+	}
+	allowed := false
+	for _, domain := range t.allowedDomains {
+		if strings.HasSuffix(u.Hostname(), domain) {
```

Comment:
isn't it risky a little bit? I mean can it allow ab.example.com for allowed b.example.com?

Reply:
It's a bit risky, however I see no way around for allowing case like example.org when remark42.example.com is used for Remark42 installation for comments on example.com. I think the solution here is outlining that risk in the documentation.

Also, the current case allowed requests to notexample.org if my domain is example.org (e.g. suffixes match): I've fixed that.

Reply:
Any other behaviour would be a breaking change. It's a small part of functionality so maybe a requirement for listing your domain(s) explicitly in AllowedDomains in case Remark42 runs on a separate domain is not bad.

I can alter the logic to get second-level domain only from `REMARK_URL` and use `ALLOWED_HOSTS` as-is, that seem to make sense.